app/javascript/article-form/components/TagsField.jsx

```diff
@@ -12,6 +12,7 @@ export const TagsField = ({ defaultValue, onInput, switchHelpContext }) => {
         onFocus={switchHelpContext}
         classPrefix="crayons-article-form"
         fieldClassName="crayons-textfield crayons-textfield--ghost ff-accent"
+        pattern="[\x30-\x39\x41-\x5A\x61-\x7A\x2C\x20]+"
```

Comment:
do you think we could save this pattern in a self explanatory constant? Or at least list in a comment which chars are restricted by these. Same for the other change

Reply:
@rhymes I updated it to a more readable format, let me know what you think! I'm not sure why I used the hex values in retrospect :sweat_smile:

Reply:
ahhaha that's fine too!